package.json

```diff
@@ -38,6 +46,7 @@
     ]
   },
   "devDependencies": {
+    "cypress": "^4.5.0",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
```

Comment:
missing dependency for npm-run-all

Reply:
Yes, I agree. This should be in the deps. I was wondering why it didn't show up, then found it in the package-lock. Do you have an idea why it didn't show up?

Reply:
Good question.. maybe you forgot the `--save-dev`?

Reply:
Nothing's impossible :D